src/components/Image/docs/ImageWiringExample.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
```

Comment:
I wonder if to remove also this one (with ImageWiringExample.st.css) until we've a settings panel support?

Reply:
I guess we can remove this for now